server/controllers/overviewController.ts

```diff
@@ -87,7 +86,7 @@ export default class OverviewController {
       offencesOverview,
       nonAssociationSummary,
     ] = await Promise.all([
-      activeCaseLoadId ? this.userService.getStaffRoles(clientToken, staffId, activeCaseLoadId) : ([] as StaffRole[]),
+      this.userService.getStaffRoles(clientToken, res.locals.user),
```

Comment:
Pass in user, allows `userService.getStaffRoles` to decide whether it needs to do the API call at all